packages/backend/src/rafiki/request/asset.request.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  Asset,
+  AssetEdge,
+  AssetMutationResponse,
+  AssetsConnection
+} from '../generated/graphql'
+import { gql } from 'graphql-request'
+import { requestGQL } from '../graphql.client'
+
+export async function createAsset(code: string, scale: number): Promise<Asset> {
+  const createAssetMutation = gql`
+    mutation CreateAsset($input: CreateAssetInput!) {
```

Comment:
do we have any way of generating these somewhere, or are we stuck with using magic strings for now?

Reply:
I did not find a way but I am new to graphql staff, @raducristianpopa maybe you know if it is possible?

Reply:
There is not really a way to generate them - that's the query language. Queries and mutations can be written in a separate file (for ie: queries can be written in `queries.graphql` and mutations in `mutations.graphql`), but I think it's better to keep them where they are actually used/needed (how it's currently done).

With `graphql-codegen` we can generate the operations (query or mutation) types, so we won't have to create them ourselves and get some type safety along the way. Either way, if we have a typo in a query or a mutation (instead of `CreateAssetInput` it's `CreateAsetInput`) `graphql-codegen` will fail because it would not be able to find the bad written field/input.

Reply:
and is graphql-codegen to be integrated in this PR aswell, to provide types? Or should we consider it a follow-up

Reply:
added a separate issue for looking into this https://github.com/interledger/testnet/issues/173